src/iOS/Xamarin.iOS/Samples/Layers/ShowPopup/ShowPopup.cs

```diff
@@ -0,0 +1,130 @@
+{
+    [Register("ShowPopup")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Show a popup",
+        category: "Layers",
+        description: "Show predefined popups from a web map.",
+        instructions: "Tap on the features to prompt a popup that displays information about the feature. ",
+        tags: new[] { "feature", "feature layer", "popup", "web map" })]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData()]
+    public class ShowPopup : UIViewController
+    {
+        // Hold references to UI controls.
+        private MapView _myMapView;
+        private PopupViewer _popupViewer;
+
+        public ShowPopup()
+        {
+            Title = "Show popup";
+        }
+
+        private void Initialize()
+        {
+            // Load the map.
+            _myMapView.Map = new Map(new Uri("https://runtime.maps.arcgis.com/home/webmap/viewer.html?webmap=e4c6eb667e6c43b896691f10cc2f1580"));
+        }
+
+        private async void MapViewTapped(object sender, GeoViewInputEventArgs e)
+        {
+            try
+            {
+                // Get the feature layer from the map.
+                FeatureLayer incidentLayer = _myMapView.Map.OperationalLayers.First() as FeatureLayer;
+
+                // Identify the tapped on feature.
+                IdentifyLayerResult result = await _myMapView.IdentifyLayerAsync(incidentLayer, e.Position, 12, true);
+
+                if (result != null && result.Popups.Any())
+                {
+                    // Get the first popup from the identify result.
+                    Popup popup = result.Popups.First();
+
+                    // Create a new popup manager for the popup.
+                    _popupViewer.PopupManager = new PopupManager(popup);
+
+                    QueryParameters queryParams = new QueryParameters
+                    {
+                        // Set the geometry to selection envelope for selection by geometry.
+                        Geometry = new Envelope((MapPoint)popup.GeoElement.Geometry, 6, 6)
+                    };
+
+                    // Select the features based on query parameters defined above.
+                    await incidentLayer.SelectFeaturesAsync(queryParams, SelectionMode.New);
+                }
+            }
+            catch (Exception ex)
+            {
+                System.Diagnostics.Debug.Print(ex.Message);
+            }
+        }
+
+        public override void LoadView()
+        {
+            // Create the views.
+            View = new UIView { BackgroundColor = ApplicationTheme.BackgroundColor };
+
+            _myMapView = new MapView() { TranslatesAutoresizingMaskIntoConstraints = false };
+            _popupViewer = new PopupViewer() { TranslatesAutoresizingMaskIntoConstraints = false };
+
+            // Add the views.
+            View.AddSubviews(_myMapView, _popupViewer);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]{
```

Comment:
This sample would be a good candidate for a responsive layout, showing the views side-by-side in landscape. A StackPanel would make that more convenient, since you should be able to just change the orientation of the panel.